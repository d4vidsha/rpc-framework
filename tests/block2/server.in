init 6000
register sleep sleep
register add2 add2
serve
