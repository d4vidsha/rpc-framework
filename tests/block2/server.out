rpc_init_server: instance 0, port 6000
rpc_register: instance 0, sleep (handler) as sleep
rpc_register: instance 0, add2 (handler) as add2
rpc_serve_all: instance 0
handler sleep2: before, 4 seconds
handler sleep2: before, 1 seconds
handler sleep2: after, 1 seconds
handler add2_i8: arguments 1 and 1
handler sleep2: after, 4 seconds
